### DISCUSSION TITLE
Documentation suggestion: Remove "(1)" from web tabs

### DISCUSSION THREAD

**Discussion by eriksjolund** [containers/podman#13438] [Ideas]
This is just a minor annoyance.

When I have opened a documentation page such as
https://docs.podman.io/en/latest/markdown/podman-run.1.html
I often get fooled that there is some update available because
the web browser tab displays "podman-run (1)".

In different web sites I've seen `(1)` as being used as an indication  that updates are available, for instance in GitHub issues.

**Comment by Luap99** [comment]
https://github.com/containers/podman/blob/f4d6e8777213880204ccbce92201c47c74b33036/docs/source/conf.py#L75
I think you would need to change this line.

**Comment by eriksjolund** [comment]
Draft PR https://github.com/containers/podman/pull/13440
